app/src/main/res/layout/fragment_register_participant_participant_details.xml

```diff
@@ -560,22 +561,37 @@
                 android:gravity="center_vertical"
                 android:orientation="horizontal">
 
-                <TextView
-                    android:id="@+id/label_gender"
-                    android:layout_width="wrap_content"
+                <LinearLayout
+                    android:layout_width="155dp"
                     android:layout_height="wrap_content"
-                    android:maxWidth="155dp"
-                    android:maxLines="1"
-                    android:minWidth="155dp"
-                    android:text="@string/participant_registration_details_label_gender"
-                    android:textAppearance="@style/LabelText" />
+                    android:orientation="horizontal">
+
+                    <TextView
+                        android:id="@+id/label_gender"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:maxLines="1"
+                        android:text="@string/label_gender_with_asterisk"
```

Comment:
please dont change this line and use participant_registration_details_label_gender as it was before